media-fonts/lxgw-wenkai/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>rtgiskard@gmail.com</email>
+		<name>giskard</name>
```

Comment:
Please provide full name.

Reply:
This is not really required...but surely a nice thing.